service/src/main/java/bio/terra/pipelines/service/JobsService.java

```diff
@@ -23,56 +27,60 @@ public class JobsService {
   private static final Logger logger = LoggerFactory.getLogger(JobsService.class);
   private final JobsRepository jobsRepository;
   private final PipelineInputsRepository pipelineInputsRepository;
+  private final StairwayJobService stairwayJobService;
 
   @Autowired
   public JobsService(
-      JobsRepository jobsRepository, PipelineInputsRepository pipelineInputsRepository) {
+      JobsRepository jobsRepository,
+      PipelineInputsRepository pipelineInputsRepository,
+      StairwayJobService stairwayJobService) {
     this.jobsRepository = jobsRepository;
     this.pipelineInputsRepository = pipelineInputsRepository;
+    this.stairwayJobService = stairwayJobService;
   }
 
   /**
-   * Creates a new pipeline service job based on a user's request. Returns jobId of new job if write
-   * to db is successful, otherwise returns null.
+   * Creates a new pipeline service job, using a Stairway flight, based on a user's request. Returns
+   * jobId of new job (which is the same as the flightId) if flight submission is successful.
    *
    * @param userId
    * @param pipelineId
    * @param pipelineVersion
-   * @return UUID jobId
+   * @return String jobId
    *     <p>Note that the information in the requested job will grow over time, along with the
    *     following related classes:
    * @see Job
    */
   @Transactional
-  public UUID createJob(
+  public String createJob(
       String userId, String pipelineId, String pipelineVersion, Object pipelineInputs) {
-    Instant timeSubmitted = getCurrentTimestamp();
-
     logger.info("Create new {} version {} job for user {}", pipelineId, pipelineVersion, userId);
 
-    // placeholder for actually doing something; for now we're just writing the info to the database
-    //        JobBuilder createJob = jobService ...
-
-    String status = "SUBMITTED";
-
-    return writeJobToDb(
-        userId, pipelineId, pipelineVersion, timeSubmitted, status, pipelineInputs, 1);
+    StairwayJobBuilder stairwayJobBuilder =
```

Comment:
is this where we would want to distinguish what stairwayjob we're created based on pipelineId or would that happen.  like would the `.flightClass()` value be dependent on the piplineId?

Reply:
oh interesting, so if the flight fails to initialize, then there will be nothing written to the tsps database.  I feel like maybe this should happen before we launch the flight?  We should chat about what makes sense here.

Reply:
yeah exactly, I think we probably want to write some stuff to the db right away. but I think that those decisions should be mapped out in [TSPS-125](https://broadworkbench.atlassian.net/browse/TSPS-125) and don't need to be solidified for this PR necessarily

Reply:
👍